src/sql/parts/jobManagement/views/jobHistory.component.ts

```diff
@@ -123,6 +123,18 @@ export class JobHistoryComponent extends Disposable implements OnInit {
 			filter: this._treeFilter,
 			renderer: this._treeRenderer
 		}, {verticalScrollMode: ScrollbarVisibility.Visible});
+		let overview: HTMLElement = $('.overview-tab').get(0);
+		let monacoShell: HTMLElement = $('.monaco-shell').get(0);
+		overview.onkeydown = (e) => {
```

Comment:
What are you trying to achieve here? Whats the current functional behavior you are trying to prevent?

Reply:
What server does this repro on? I can't seem to repro